providers/gce/gce_annotations_test.go

```diff
@@ -0,0 +1,73 @@
+// +build !providerless
```

Comment:
providerless is a build directive specific to K/K. (See https://github.com/kubernetes/kubernetes/pull/80353) It may make sense to keep it for now, as it may still be useful when we are vendored back into K/K but we should have an issue to clean this up once extraction completes. Also its worth being aware of as if we somehow inherit those build rules into this repo it will cause us problems.

Reply:
Thanks for the info. Created the issue for tracking: https://github.com/kubernetes/cloud-provider-gcp/issues/210